src/connectors/captcha.ts

```diff
@@ -40,15 +41,47 @@ function start() {
         parentOrigin = new URL(parentUrl).origin;
     }
 
+    let decodedData: any;
+    try {
+        decodedData = JSON.parse(b64Decode(data));
+    }
+    catch (e) {
+        error('Cannot parse data.');
+        return;
+    }
+    callbackUri = decodedData.callbackUri;
+
+    let src = 'https://hcaptcha.com/1/api.js?render=explicit';
+
+    // Set language code
+    if (decodedData.locale) {
+        src += `&hl=${decodedData.locale ?? 'en'}`;
```

Comment:
Do we need to process this in any way? Do hcaptcha support all locales.

Reply:
If hcaptcha can't find the locale we request it uses it's default, which attempts to detect the appropriate locale. The only reason I'm doing this is to try and match what you're already specified to bitwarden.